.github/workflows/image.yaml

```diff
@@ -0,0 +1,36 @@
+name: Docker image
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  # Build image and deploy to ghcr.io
+  docker:
+    name: Build and deploy Docker image
+    runs-on: ubuntu-latest
+    env:
+      IMAGE: ghcr.io/toothstone/cert-manager-webhook-anexia
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Get the version
+        run: echo "VERSION=${GITHUB_REF/refs\/tags\/v/}" >> $GITHUB_ENV
+      - name: Login
+        run: |
+          echo $PAT | docker login ghcr.io --username toothstone --password-stdin
```

Comment:
toothstone again

Reply:
In this case this will actually stay like this, or we need to get a PAT from a bot account for the anexia-it org

Reply:
I asked around, there is no bot account for anexia-it. So for now my PAT will be used here.